### PR TITLE
Integrate map

### PR DESCRIPTION
**Pull request by tommysr**

### CODE REVIEW

File: app/src/routes/shipmentsMap/bought/+page.svelte [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+	$: if ($walletStore.publicKey) {
+		searchableBoughtShipments.update((s) => {
+			s.filtered = s.data.filter((s) => s.account.buyer === $walletStore.publicKey?.toString());
+
+			s.data = s.filtered;
+
+			return s;
+		});
+	} else {
+	}
```
Comment (matzayonc) [inline app/src/routes/shipmentsMap/bought/+page.svelte:26]:
```suggestion
	}
```

**Review by matzayonc — APPROVED** [review]

File: app/src/routes/shipmentsMap/bought/+page.svelte (hunk 1 above)
Comment (tommysr) [reply app/src/routes/shipmentsMap/bought/+page.svelte:26]: breathtaking

File: app/src/components/Shipment/BoughtOrderListElement.svelte
Comment (tejks) [inline app/src/components/Shipment/BoughtOrderListElement.svelte:1]: Duplicate code. To be changed in the future.

**Review by tejks — APPROVED** [review]

File: app/src/components/Shipment/BoughtOrderListElement.svelte
Comment (tommysr) [reply app/src/components/Shipment/BoughtOrderListElement.svelte:1]: agreed

**Pull request merged**